### ISSUE TITLE
Change font path via mixins.scss

### ISSUE THREAD

**Issue by lonix1** [fontsource/fontsource#143]
## **Is your feature request related to a problem? Please describe.**
I need to change the base path from `./files/`. I assume I must use [`mixins.scss`](https://github.com/fontsource/fontsource/blob/master/scripts/templates/scss.js#L28) for that.

But I can't do this:
````scss
@include fontFaceCustom($fontDir: '/static/');
````

I must instead override the whole thing like this:
````scss
@include fontFaceCustom(
  $woff2Path: "/static/montserrat-latin-400-normal.woff2",
  $woffPath:  "/static/montserrat-latin-400-normal.woff"
);
````

## **Describe the solution you'd like**
I'm using `fontFaceCustom()` rather than `fontFace()`, because I don't want the unicode stuff.

So:
- the `fontFaceCustom()` mixin should also provide a `$fontDir` variable
- OR, the `fontFace()` mixin should also provide a `$unicodeRange: false` option

Thanks for this great tool!

**Comment by ayuhito** [comment]
Thanks for the issue!

Do you mean we should adjust the default mixin to be like this?

```scss
@mixin fontFaceCustom(
  ...
  $fontDir: "~@fontsource/#{$fontId}/files",
  $woff2Path: "#{$fontDir}/#{$fontId}-<%= defSubset %>-#{$weight}-#{$style}.woff2",
  $woffPath: "#{$fontDir}/#{$fontId}-<%= defSubset %>-#{$weight}-#{$style}.woff",
  ...
 )
```
 
 At least this change shouldn't cause any breaking changes I think? Whilst allowing the flexibility of just changing fontDir.

**Comment by lonix1** [comment]
Yes - and it won't cause breaking changes. It would allow us to override the base path for both mixins. Double flexibility.

So if you want the unicode stuff you use `fontFace`, and if you don't then you use `fontFaceCustom`.

**Comment by ayuhito** [comment]
> So if you want the unicode stuff you use fontFace, and if you don't then you use fontFaceCustom.

Yeah, fontFaceCustom is pretty much a copy of fontFace without the whole unicode-map feature. I need to account for fonts with tons of subsets like [this](https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@400) after all.

**Comment by lonix1** [comment]
At some point it may be a good idea to just consolidate them into one mixin. And use a variable to let the user opt-in/out of the unicode stuff. Unsure if that's feasible, but would be neat.

Something like
````scss
@if $unicodeRange {
  // stuff from fontFace()
} @else {
  // etc.
}
````

That would also be backwards-compatible, and then you could deprecate the `fontFaceCustom` mixin at a later date.

**Comment by ayuhito** [comment]
Initial suggestion released in 4.2.0.

> At some point it may be a good idea to just consolidate them into one mixin. And use a variable to let the user opt-in/out of the unicode stuff. Unsure if that's feasible, but would be neat.
> 
> Something like
> 
> ```css-scss
> @if $unicodeRange {
>   // stuff from fontFace()
> } @else {
>   // etc.
> }
> ```
> 
> That would also be backwards-compatible, and then you could deprecate the `fontFaceCustom` mixin at a later date.

Both mixins have different usecases which can't be merged, so I'm not too in favour that suggestion. Thanks for the issue though!

**Comment by lonix1** [comment]
Thanks for that update.

As far as the merging goes - yes I see now there are differences between them other than unicode.
